gatsby/sourceplugins.rst

```diff
@@ -19,17 +19,19 @@ After that, we need to enable the plugin in our project. To do this, we need to
 
   ...
   plugins: [
+    ...
     {
       resolve: `gatsby-source-filesystem`,
       options: {
         name: `src`,
         path: `${__dirname}/src/`,
       },
-    }
+    },
+    ...
   ]
   ...
 
-Now restart the development server. If you open GraphiQL page, you'll see new possible queries.
+Now restart the development server. If we open GraphiQL page, we will see new possible queries.
```

Comment:
One sentence per line.

s/GraphiQL/GraphQL

Reply:
No..it's really GraphiQL ;) it's a tool to test and inspect GraphQL queries.
We talked about it in data chapter

Reply:
@cekk as suggestion, I would use GraphiQL then as link back to the anchor of it in the data chapter :)

Reply:
OK, I thought it was a typo. In that case, either @svx's suggestion or add a term to the Glossary, whichever makes more sense in this context.